includes/admin/class-admin.php

```diff
@@ -1162,6 +1179,42 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'install_plugins' ) || ! WPBDP_App_Helper::is_bd_page() ) {
+				return;
+			}
+
+			if ( ! wpbdp_get_option( 'enqueue-fontawesome-styles', false ) ) {
+				return;
+			}
+
+			if ( is_plugin_active( 'font-awesome/index.php' ) ) {
+				$this->messages[] = array(
+					__( 'Good news! Business Directory Plugin now integrates with the official Font Awesome plugin.', 'business-directory-plugin' ),
```

Comment:
This message doesn't look very helpful to me. Let's remove it.

Reply:
Message removed